docs/how-tos/consume-an-event.rst

```diff
@@ -0,0 +1,133 @@
+Consume an Open edX Event
+=========================
+
+You have two ways of consuming an Open edX event, within the same service or in a different service. In this guide, we will show you how to consume an event within the same service. For consuming events across services, see :doc:`../how-tos/use-the-event-bus-to-broadcast-and-consume-events`.
+
+.. note:: We encourage you to also consider the practices outlined in the :doc:`../decisions/0016-event-design-practices` ADR for event consumption.
+
+Throughout this guide, we will implement the use case to send the enrollment data to a webhook when a user enrolls in a course to better illustrate the steps involved in creating a consumer for an event.
+
+Assumptions
+-----------
+
+- You have a development environment set up using `Tutor`_.
+- You have a basic understanding of Python and Django.
+- You have created a new Open edX event. If not, you can follow the :doc:`../how-tos/create-a-new-event` guide to create a new event.
```

Comment:
We may want to edit this part because we don't need a new event to consume events.

Reply:
Thanks for noticing! https://github.com/openedx/openedx-events/pull/439/commits/7980afc14fc802f9da6af0736ea3ae698081243f